### DISCUSSION TITLE
Problem when the page reload content in svelte

### DISCUSSION THREAD

**Discussion by daniortlepp** [frontendbr/forum#1658] [Perguntas e Respostas]
I have a problem. After the user logs in, they are redirected to a page where I load a menu and other content. When you log off, you are redirected back to the login, but the menu still appears, and should not. It looks like it only prints content on the page, it doesn't load it all. Does anyone know how I can solve this? I'm a beginner.

**Comment by felipefialho** [comment]
Just in portuguese here, please.

(Apenas em português aqui, por favor)

**Comment by daniortlepp** [comment]
Desculpe.
Estou com um problema. Após o usuário fazer login, é redirecionado para uma página onde um menu é carregado. Quando faz logoff, é redirecionado de volta para o login, porém o menu continua aparecendo, sendo que não deveria e nem chamo ele na página do login. Parece que só imprime o conteúdo na página e não carrega por completo. Alguém sabe como resolver isso? Eu sou iniciante no Svelte.

**Comment by ninetails** [comment]
Teria como fazer um [Code Sandbox](https://codesandbox.io/) com a parte necessária, mockando o login e o logout?
